pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ClientConfigurationData.java

```diff
@@ -58,6 +58,9 @@
     private int maxLookupRequest = 50000;
     private int maxNumberOfRejectedRequestPerConnection = 50;
     private int keepAliveIntervalSeconds = 30;
+
+    private long connectTimeOutSeconds = 60;
```

Comment:
Why add these to `ClientConfigurationData` when they're only relevant to admin client?